app/src/main/java/com/testtask/apptesttask/presentation/characters/CharactersPresenter.kt

```diff
@@ -1,15 +1,39 @@
 package com.testtask.apptesttask.presentation.characters
 
 import com.arellomobile.mvp.InjectViewState
+import com.testtask.apptesttask.model.interactor.characters.CharactersInteractor
 import com.testtask.apptesttask.presentation.base.BasePresenter
+import io.reactivex.android.schedulers.AndroidSchedulers
+import io.reactivex.schedulers.Schedulers
+import javax.inject.Inject
 
 @InjectViewState
-class CharactersPresenter :
+class CharactersPresenter @Inject constructor(private val charactersInteractor: CharactersInteractor) :
     BasePresenter<CharactersView>() {
 
     override fun onFirstViewAttach() {
         super.onFirstViewAttach()
 
-        // TODO: characters tab (Add view implementation).
+        loadCharacters()
+    }
+
+    fun loadCharacters() {
+        unsubscribeOnDestroy(charactersInteractor.getCharacters()
+                .subscribeOn(Schedulers.io())
```

Comment:
We discussed the schedulers logic two days before. Create `SchedulersProvider` as it done in `GitFox`.